examples/host_filter_dag.json

```diff
@@ -38,6 +39,16 @@
     },
     {
       "in": ["star_out"],
+      "out": "trimmomatic_out",
```

Comment:
why do we do this after star?

Reply:
Because STAR has the efficient way of incorporating the truncation behavior (max 75 million)... I didn't think it was worth it to change that. WDYT?

Reply:
sure. but i think we already truncated at the web level and we can also do the max_reads at the given target level?